src/tchannel_bridge.js

```diff
@@ -76,8 +78,7 @@ export default class TChannelBridge {
                 span.setTag('as', request.headers.as);
             }
 
-            // In theory may overwrite tchannel span, but thats what we want anyway.
-            context.openTracingSpan = span;
+            context.setSpan(span);
 
             // remove headers prefixed with $tracing$
             let headerKeys = Object.keys(headers);
```

Comment:
L93: `handlerFunc(context, request, headers, body, wrappingCallback);`

Aren't you making it backwards incompatible? Didn't we say it should be this?
```
request.context = context;
handlerFunc(context, request, headers, body, wrappingCallback);
```